pipe/parQuantumLine/parQuantumLine_group_loader.yaml

```diff
@@ -9,14 +9,15 @@ transform:
     OUT_PATH: /pfs/out    
     # ERR_PATH can be changed, it is user specified
     ERR_PATH: /pfs/out/errored_datums
-  image: quay.io/battelleecology/group_loader:68b73fdc
+  image: us-central1-docker.pkg.dev/neon-shared-service/neonscience/neon-is-group-loader:sha-a900e1e
   image_pull_secrets:
     - battelleecology-quay-read-all-pull-secret
   secrets:
   - name: pdr-secret
     mount_path: /var/db_secret
   stdin:
     - '#!/bin/bash'
+    - cd modules/
```

Comment:
Why does it need to cd into modules? Better to have the code accessible from the working directory of the image. Looks like I missed that change to the Dockerfile - sorry for not catching it sooner. Please adjust the Dockerfile back to placing the code directly into the working directory (not nested within modules/).